requirements.txt

```diff
@@ -12,5 +12,6 @@ toml>=0.10.2,<0.11
 packaging>=21,<23
 omegaconf>=2.2.3,<3
 # these are just for common/text_data.py
+datasets>=2.9.0
 transformers>=4.11,<5
```

Comment:
The file header says 
> These requirements are shared for _**all**_ examples

Should 
```
datasets>=2.9.0
transformers>=4.11,<5
```
be part of this req file since they are not needed for any of the cv examples

Reply:
hmmm... shared but only for text...

Reply:
**_ALL_**

lol.
yeah I guess this'll be @dblalock's call

Reply:
they should probably be in both llm and bert, but with the same versions.

Reply:
i'm going to leave them here for now and create a task to fix it. fixing it means also gating the transformers/datasets imports in common.